troposphere/static/js/components/settings/AdvancedSettingsPage.jsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
+
 import SSHConfiguration from "components/settings/advanced/SSHConfiguration";
+import ClientCredentials from "components/settings/advanced/ClientCredentials";
```

Comment:
Note that ExportCredential.jsx is already in undulating-umbrellabird:

https://github.com/cyverse/troposphere/blob/undulating-umbrellabird/troposphere/static/js/components/modals/ExportCredential.jsx